.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+.idea
```

Comment:
Personally, I feel like this should be handled by a user's global git ignore rather than the project's.